CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ if(EXISTS ${LOC_PATH})
     message(FATAL_ERROR "You cannot build in the source directory (or any directory with a CMakeLists.txt file). Please make a build subdirectory.")
 endif()
 
+set(OR_GROUPSIZE 128)
+set(OR_GRAINSIZE 7)
```

Comment:
Ahh, I see.  Do you recommend that I make a 'ThrustOverride.h.in' to set these values?  

Maybe it would be better to modify these two lines as follows?
add_definitions ("-DOR_GROUPSIZE 128")
add_definitions ("-DOR_GRAINSIZE 128")

Either way, the code will need to be recompiled fully.